packages/wrangler/e2e/helpers/normalize.ts

```diff
@@ -62,12 +63,19 @@ function removeTimestamp(str: string) {
 		.replace(/\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d\.\d+?Z/g, "TIMESTAMP")
 		.replace(/\d\d:\d\d:\d\d/g, "TIMESTAMP");
 }
+
 function removeUUID(str: string) {
 	return str.replace(
 		/\w{8}-\w{4}-\w{4}-\w{4}-\w{12}/g,
 		"00000000-0000-0000-0000-000000000000"
 	);
 }
+function removeBinding(str: string) {
```

Comment:
I'd like some sanity check on this fn pls 🙏

Reply:
here is how we create that binding name: https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/utils/getValidBindingName.ts

Reply:
the UUID contains dashes `-`, whereas the binding name contains underscores `_`, so `removeUUID` won't have any effect on the binding name :(

Reply:
looking at https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/utils/getValidBindingName.ts#L11 and https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/utils/getValidBindingName.ts#L13 particularly, I wonder if my implementation is actually correct, since my assumption on the number of caracters separated by dashes might not be correct

Reply:
I think the point is that R2 is actually using a UUID in their name but we are converting it to a valid JS identifier, so it does have the same number of characters as a UUID, but the dashes are converted to underscores.

And also this is just for tests, so not too deep; if it breaks in the future we can just fix it again.

Reply:
I did double check and this is indeed used only in the e2e tests. Thank you for reviewing Pete <3